File: javascript/height-checker.js
function heightChecker(height) {
  let expected = [...height].sort((a, b) => a - b);
  let mismatch = 0;
  for (let i = 0; i < height.length; i++) {
    if (height[i] !== expected[i]) {
      mismatch++;
    }
  }
  return mismatch;
}

// --- Input dari terminal ---
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Masukkan array tinggi (pisahkan dengan spasi): ", (answer) => {
  let height = answer.trim().split(/\s+/).map(Number);
  let result = heightChecker(height);
  console.log("Jumlah siswa yang tidak sesuai urutan:", result);
  rl.close();
});

File: python/height-checker.py
def heightChecker(height):
    expected = sorted(height)
    mismatch = 0
    for h, e in zip(height, expected):
        if h != e:
            mismatch += 1
    return mismatch

if __name__ == "__main__":
    # User masukkan array di terminal
    raw_input = input("Masukkan array tinggi (pisahkan dengan spasi): ")
    height = list(map(int, raw_input.strip().split()))
    
    result = heightChecker(height)
    print("Jumlah siswa yang tidak sesuai urutan:", result)
